geonode/upload/admin.py

```diff
@@ -59,5 +62,15 @@ def has_delete_permission(self, request, obj=None):
         return super(UploadSizeLimitAdmin, self).has_delete_permission(request, obj)
 
 
+class UploadParallelismLimitAdmin(admin.ModelAdmin):
+    list_display = ('slug', 'description', 'max_number',)
+
+    def has_delete_permission(self, request, obj=None):
+        if obj and obj.slug == "default_max_parallel_uploads":
```

Comment:
Via the admin interface, the `default_max_parallel_uploads` is not available for the user to change the value because it will be created at the first upload. Adding it to the fixture could fix this problem.
Plus I'll suggest removing the possibility to add new parallelism since they will not have any effect, should be enough to do:
```python
def has_add_permission(self, request):
    return False
```

Reply:
Updated